happi/client.py

```diff
@@ -174,7 +162,7 @@ def add_device(self, device):
         logger.info('Device %r has been succesfully added to the '
                     'database', device)
 
-    def load_device(self, **post):
+    def find_device(self, **post):
```

Comment:
If you want to make it really explicit that this looks up a database entry and doesn't actually return a `Device` subclass, you can pick a name more like `find_entry` or `find_data`, or something similar.

Reply:
Yeah the awkward part is that I have a `find_document` method that returns the raw dictionary information. We can argue how useful that is ...